UnityProject/Assets/Scripts/Objects/Alien/AlienEggCycle.cs

```diff
@@ -189,6 +213,10 @@ private void FeelsSlimy(Interaction interaction)
 				$"{interaction.Performer.ExpensiveName()} touches the slimy egg!");
 		}
 
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			return !(interaction.Intent == Intent.Harm && currentState == EggState.Grown);
```

Comment:
My bad, I thought this was already a checked interactable. You need to include the default check (see other checked interactables for this)

Reply:
Yeah, now I see I forgot that default check, but it should be good in my last commit.